mithril.js

```diff
@@ -691,7 +704,11 @@ var coreRenderer = function($window) {
 		if (recycling) {
 			initComponent(vnode, hooks)
 		} else {
-			vnode.instance = Vnode.normalize(vnode._state.view.call(vnode.state, vnode))
+			try {
+				vnode.instance = Vnode.normalize(vnode._state.view.call(vnode.state, vnode))
+			} catch (err) {
+				console.error(err)
```

Comment:
Please reset `vnode.instance` here to `undefined`, since it starts out with the old tree + state.